docs/tapable-hooks.md

```diff
@@ -231,37 +236,38 @@ const serverHooks: ViseHooks = {
     return result;
   },
 
-  // 在 HTML 缓存命中后并行执行所有钩子，然后响应 HTTP 请求，无法在此更改响应，可做统计等
+  // Tapped functions will be notified a successful cache hit event.
```

Comment:
Tapped functions will be notified with a successful cache hit event.

Reply:
added `with`
fixed